man/systemd.network.xml

```diff
@@ -594,6 +594,17 @@
         </para></listitem>
         </varlistentry>
         <varlistentry>
+          <term><varname>ProxyNDP=</varname></term>
```

Comment:
But does not Neighbor Discovery Protocol already imply ipv6? I mean it there is also no prefix for Proxy ARP.